recipe_scrapers/wholefoods.py

```diff
@@ -0,0 +1,7 @@
+from ._abstract import AbstractScraper
+
+
+class WholeFoods(AbstractScraper):
```

Comment:
Would you mind adding the other accessor methods to this? I previously thought we had a style preference, but couldn't find it. Now, I remembered that there was a commit (80fd5e3bc20062ffbf06386feb42924f4e16eac0) where we previously back filled in a lot of these methods.

Reply:
Added per provided commit.  Thank you.  I will follow this pattern in the future.